prime-angular-frontend/src/app/shared/components/college-certification-form/college-certification-form.component.ts

```diff
@@ -97,6 +98,19 @@ export class CollegeCertificationFormComponent implements OnInit {
     this.remove.emit(this.index);
   }
 
+  public shouldShowPractices(): boolean {
+    if ((this.collegeCode.value == CollegeLicenceClass.BCCNM) &&
+        (this.licenseCode.value == NursingLicenseCode.NON_PRACTICING_REGISTERED_NURSE ||
```

Comment:
Feels like the set of OR'd values could be reduced down to an array of enum values and use the include method to check if the value exists in the array - `[...].includes(value)`